src/pages/EmployeeDetail/components/EmployeeForm.tsx

```diff
@@ -47,6 +54,30 @@ export default function EmployeeForm(props: EmployeeFormProps) {
         innerRef={formik}
       >
         <Form id="emp-form" noValidate>
+          <div className={"profile-upload " + (props.isView ? "is-view" : "")}>
+            <label htmlFor="imageUpload" tabIndex={0}>
+              <img src={profileImg} alt="employee profile" />
+              <div className={props.isView ? "is-view" : ""}>
+                <span className="material-symbols-outlined"> edit </span>
+              </div>
+              <input
+                id="imageUpload"
+                style={{ display: "none" }}
+                type="file"
+                name="employee profile"
+                onChange={(e) => {
+                  if (e.target.files) {
```

Comment:
use a method and move method commands there like onChange={handleOnChangeEvent}

Reply:
Yes, will do that